cds/modules/deposit/api.py

```diff
@@ -106,7 +106,6 @@ def _dumps(obj):
                 'size': obj.file.size if obj.file else 0,
                 'file_id': str(obj.file_id),
                 'completed': obj.file is not None,
-                'progress': 100 if obj.file is not None else '',
```

Comment:
We merge the object we get from `update_deposit` events to the deposit state, and this caused the actual progress of the subformats to be overwritten.